src/coordinator.js

```diff
@@ -5,7 +5,21 @@ import './environment';
 import logger from './utils/logger';
 import { raceTo } from './utils/lifecycle';
 
-const WORKER_COUNT = os.cpus().length - 1 || 1;
+export function getWorkerCount(getCPUs) {
+  const realCount = os.cpus().length;
+
+  if (!getCPUs) {
```

Comment:
i think instead we should define a function `getDefaultCPUs` in the module scope, and do `getCPUs = getDefaultCPUs` in the function signature.